master/getting-started/openshift/installation.md

```diff
@@ -0,0 +1,87 @@
+---
+title: Installing Calico on OpenShift
+---
+
+Installation of Calico in OpenShift is integrated in openshift-ansible v3.6.
+
+To enable an installation of Calico that shares the etcd
```

Comment:
I do think this section still needs a heading if nothing else to denote that it and "Bring-your-own" are two exclusive sections.